components/list/list.js

```diff
@@ -0,0 +1,60 @@
+// Imports
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * General List component. Use when you need to display array of elements
+ */
+function List({ items, hideBullets, mods = [], align }) {
+  mods.push(`align_${align}`);
```

Comment:
@fahad19 (_correct me if I am wrong_ ) As I remember the problem was when having sibling components and passing object/array their rendering was affected from their sister components. Also what i remember at that time the problem was solved by defining default empty array/object for each component and then populating it with elements/properties.